packages/form/addon/components/cf-field.hbs

```diff
@@ -14,7 +14,10 @@
 
     <div class="uk-flex">
       <div class="uk-width-expand">
-        {{#let (component (get-widget @field.question)) as |FieldComponent|}}
+        {{#let
+          (component (ensure-safe-component (get-widget @field.question)))
```

Comment:
Is `ensure-safe-component` still needed, since you already cover it in the `get-widget` helper?

Reply:
Yes, because we use it in a `component` helper.

Reply:
Alright. Thanks for clarification. :+1: